accounts-guest-server.js

```diff
@@ -10,6 +10,24 @@ Accounts.removeOldGuests = function (before) {
     return res;
 };
 
+/**
+ *  drop guest when visitor logs in
+ *
+ */
+GuestUsers = new Mongo.Collection('guestUsers');
+Accounts.onLogin(function(par){
+    if(par.user.username.indexOf('guest') !== -1){
+        if(!GuestUsers.findOne({connection_id: par.connection.id})){
+            GuestUsers.insert({connection_id: par.connection.id, user_id: par.user._id});
+        }
+    }
+    else if(par.type !== 'resume'){
```

Comment:
I'm not familiar with `par.type` and I'd like to know in what case would it be 'resume'? The official doc only says 'The service name, such as "password" or "twitter".'. What are the possible values?